react-cors-spa/yarn.lock

```diff
@@ -2613,6 +2613,15 @@ axe-core@^4.4.3:
   resolved "https://registry.npmjs.org/axe-core/-/axe-core-4.5.1.tgz"
   integrity sha512-1exVbW0X1O/HSr/WMwnaweyqcWOgZgLiVxdLG34pvSQk4NlYQr9OUy0JLwuhFfuVNQzzqgH57eYzkFBCb3bIsQ==
 
+axios@^1.7.5:
```

Comment:
Why do we have a yarn lock file?

Reply:
It's the base package installer/manager for the react-cors-spa repo, forgot to remove this!